pages/encrypt/index.jsx

```diff
@@ -44,7 +43,7 @@ module.exports = React.createClass({
         </head>
         <body>
           <div id="my-app">
-            <Page>{this.props.children}</Page>
+            {this.props.children}
```

Comment:
This makes sense, when I tried to do it, it wasn't working, but that may have been a result of other bugs...